packages/marketplace/contracts/Exchange.sol

```diff
@@ -112,6 +112,12 @@ contract Exchange is
         _setOrderValidatorContract(contractAddress);
     }
 
+    /// @notice Set the LAND contract address.
+    /// @param contractAddress New LAND contract address.
+    function setLandContract(ILandToken contractAddress) external onlyRole(DEFAULT_ADMIN_ROLE) {
+        _setLandContract(contractAddress);
+    }
+
```

Comment:
I don't think we need to set the land contract address, we could just send the address as an input as we do for any token; and check for the quad interface presence on the address

Reply:
I feel that having the address as an input is riskier / more inefficient than just setting the address...